### PR TITLE
📊 snapshot for Guadamuz et al. statin utilization paper

### PR DESCRIPTION
**Pull request by veronikasamborska1994**

### CODE REVIEW

File: etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# NOTE: To learn more about the fields, hover over their names.
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml:1]: Processing level is not included and it's required

File: etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        title: Statin inclusion on essential medicine list
+        unit: arbitrary units
+        short_unit: au
+        description_short: Denotes whether statins were included in national Essential Medicines Lists (EMLs).
+        display:
+          numDecimalPlaces: 0
+
+      statin_use:
+        title: Statin use
+        unit: "per 1000 population ≥ 40 years old"
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml:17]: `DDD (defined daily doses)` per 1000 population...

File: etl/steps/data/meadow/statins/2023-10-05/bmj_2022.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    # Melting the 'statin_utilization' columns and extracting the year
+    melted_statin_utilization = pd.melt(
+        tb,
+        id_vars=["country"],
+        value_vars=["statin_utilization_october_2019", "statin_utilization_september_2020"],
+        var_name="indicator",
+        value_name="value",
+    )
+    melted_statin_utilization["year"] = melted_statin_utilization["indicator"].str.extract(r"(\d+)")
```
Comment (paarriagadap) [inline etl/steps/data/meadow/statins/2023-10-05/bmj_2022.py:34]: I think you need to address the pre-COVID data includes 03-2020

File: etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        short_unit: ""
+        description_short: Based on purchases made by stores and hospitals in 52 countries and, in special cases like the UK, using data about the volume given to patients.
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml:19]: Maybe you can include a text in `description_processing` mentioning that data for this variable in 2019 also includes 2020

File: etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# NOTE: To learn more about the fields, hover over their names.
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml:1]: Processing level is not included and it's required

File: etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+tables:
+  lancet_2022:
+    variables:
+      statin_use_secondary:
+        title: Statin use for secondary preventation amongst elibible individuals
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml:10]: *prevention, *eligible

File: etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+        unit: "%"
+        short_unit: "%"
+        description_short: Proportion of eligible individuals self-reporting use of statins for the secondary preventation of cardiovascular disease
+        description_from_producer: Eligibility for statin therapy for secondary prevention was defined among individuals with a history of self-reported cardiovascular disease.
+        display:
+          numDecimalPlaces: 1
+      statin_use_primary:
+        title: Statin use for primary preventation amongst elibible individuals
```
Comment (paarriagadap) [inline etl/steps/data/garden/statins/2023-10-05/lancet_2022.meta.yml:18]: *prevention, *eligible

File: etl/steps/data/meadow/statins/2023-10-05/bmj_2022.py [hunk 8]
```diff
@@ -0,0 +1,77 @@
+"""Load a snapshot and create a meadow dataset."""
```
Comment (paarriagadap) [inline etl/steps/data/meadow/statins/2023-10-05/bmj_2022.py:1]: There is this `SettingWithCopyWarning`, but I can't find where:
```
/Users/parriagadap/etl/lib/catalog/owid/catalog/tables.py:390: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  super().__setitem__(key, value)
```

**Review by paarriagadap — APPROVED** [review]
Looks great! Especially the pdf data extraction. I added some minor comments to look at.

File: etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml (hunk 2 above)
Comment (veronikasamborska1994) [reply etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml:17]: thanks!!

File: etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml (hunk 4 above)
Comment (veronikasamborska1994) [reply etl/steps/data/garden/statins/2023-10-05/bmj_2022.meta.yml:19]: good point! thanks!! :)

**Pull request merged**